about-us.html

```diff
@@ -55,84 +65,70 @@
     </div>
 
     </header>
-    <section class ="hero container">
-        <h1><b>KidCod</b>,
-            <br> Where Kids Learn To Code By Coding</h1>
-        <div class="text-container">
-            <p>Live Classes With Seasoned professional Instructors, Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam tenetur nisi, nostrum eius animi voluptatibus veritatis atque voluptates doloribus excepturi ea</p>
-        </div> 
-            <div class="availabity">
-                <p>We are available from:</p>
-                <div class="available-day">
-                    <h5>Monday</h5><h6>To</h6> <h5>Friday</h5>
-                </div>
+    <section class="hero container">
+            <div class="hero-text  hero-text-in-about">
+                <p>Hello sharing world</p>
+                <h1>KidCod!, best Bootcamp for kids in 2022</h1>
             </div>
-            <div class="contact-info">
-                <p>Please contact per Email any futher questions</p>
-                <p>about <b>kidCo</b></p>
-                <u>kidco@gmail.com</u>
+           
+            <div class="text-container text-container-in-about">
+                <p>Live Classes With Seasoned professional Instructors, Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam tenetur nisi
+                    , nostrum eius animi voluptatibus veritatis atque voluptates doloribus excepturi ea
+                    Live Classes With Seasoned professional Instructors, Lorem ipsum 
+                    dolor sit amet consectetur adipisicing elit. Aliquam tenetur nisi,
+                    um eius animi voluptatibus veritatis atque voluptates doloribus
+                    pturi ea, Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    quam tenetur nisi, nostrum eius animi voluptatibus veritatis atque
+                    luptates doloribus excepturi eais simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text olor sit amet consectetur adipisicing elit. Aliquam tenetur nisi,
+                    strum eius animi voluptatibus veritatis atque voluptates doloribus
+                    xcepturi ea, Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    Aliquam tenetur nisi, nostrum eius animi voluptatibus veritatis atque
+                     voluptates doloribus excepturi eais simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                </p>
+            </div> 
+            <div class="contacts-info">
+                <h3>Please contact us per email for any further questions about kidco</h3>
+                <p>kidco@gmail.com</p>
             </div>
     </section>
```

Comment:
- [ ] [OPTIONAL] For a better UI, kindly remove the excess space that appears on the landing page and the About Us page. This will grant a more presentable User Interface and make your design align with the provided template.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/58692590/159042104-b60e9061-b587-431e-af9b-3dd6186406e3.png) | ![](https://user-images.githubusercontent.com/58692590/159042336-5a196b38-6256-4862-a1a0-ae986b6a13b7.png)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/58692590/159042543-2a86b0c3-edd4-43fa-a1ad-64bf1385ab19.png) | ![](https://user-images.githubusercontent.com/58692590/159042831-d170bd14-9293-4244-852e-23389f00575f.png)